packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -794,8 +794,12 @@ export class SingleThreadedWallet extends EventEmitter<EventEmitterType>
     response.channelUpdatedEvents().forEach(event => this.emit('channelUpdated', event.value));
   }
 
-  async channelFinalized(_arg: ChannelFinalizedArg): Promise<void> {
-    return;
+  async channelFinalized(arg: ChannelFinalizedArg): Promise<void> {
```

Comment:
Out of scope for this PR, but I still find it strange that the wallet implements a ` ChainEventSubscriberInterface`. That approach causes functions such as this one to be exposed in the external API of the wallet. In my opinion this can cause confusion about which other bits of code should call such functions.

Reply:
I was hacking on this over at #3116.